lib/experimental/Navigation/Carousel/index.stories.tsx

```diff
@@ -74,10 +72,27 @@ export const AutoScroll: Story = {
     autoplay: true,
     delay: 3000,
     children: SLIDES,
+    columns: { default: 1 },
   },
 }
 
 export const SneakPeek: Story = {
+  decorators: [
+    (Story) => (
+      <div className="w-full p-6">
+        <Story />
+      </div>
+    ),
+  ],
+  args: {
+    showArrows: false,
+    children: SLIDES,
+    showPeek: true,
+    columns: { default: 1 },
```

Comment:
I've said that in another PR but I'm surprised to see that many choices here. Unless I'm missing something, this component should just show or hide arrows depending on the amount of slides, no?

Reply:
for me is ok, we just need to decide :) I can change it